modules/cli-options/src/main/scala/scala/cli/commands/ScalacOptions.scala

```diff
@@ -33,29 +32,34 @@ object ScalacOptions {
     Set("-V", "-W", "-X", "-Y")
   private val scalacOptionsPrefixes =
     Set("-g", "-language", "-opt", "-P", "-target") ++ scalacOptionsPurePrefixes
+
+  /** This includes all the scalac options which disregard inputs and print a help and/or context
+    * message instead.
+    */
+  val ScalacPrintOptions: Set[String] =
+    scalacOptionsPurePrefixes ++ Set("-help", "-Xshow-phases", "-Vphases")
+
   private val scalacOptionsArgument: Argument[List[String]] =
     new Argument[List[String]] {
 
-      val underlying = StandardArgument[List[String]](scalacOptionsArg)
+      val underlying: StandardArgument[List[String]] = StandardArgument(scalacOptionsArg)
 
-      val arg = scalacOptionsArg
+      val arg: Arg = scalacOptionsArg
 
-      def withDefaultOrigin(origin: String) = this
-      def init                              = Some(Nil)
+      def withDefaultOrigin(origin: String): Argument[List[String]] = this
+      def init: Option[List[String]]                                = Some(Nil)
       def step(
         args: List[String],
         index: Int,
         acc: Option[List[String]],
         formatter: Formatter[Name]
-      ) =
+      ): Either[(Error, List[String]), Option[(Option[List[String]], List[String])]] =
         args match {
-          case h :: t
-              if scalacOptionsPrefixes.exists(h.startsWith) &&
-              !scalacOptionsPurePrefixes.contains(h) =>
```

Comment:
This shouldn't be removed. This prevents us from using `-V` / `-W` / `-X` / `-Y` ourselves (like `-X` as an alias for `--cross`)

Reply:
Or we should remove the current use of `-X` in the code base (look for `"X"`)

Reply:
ended up removing the current use of `-X`